coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
No I meant user passes aspects. `allow` or `disallow` tastes are passed to the Bear. :smiley: Yup its the bear writer who will decide the aspects I quite agree to that but we can restrict the bear to analyze one aspect out of two right (if it analyzes two aspects of code). So we can control that with these with allow or disallow tastes. So if no aspects are passed and bear is called explicitly, if we pass Root aspect, `allow` value will be true for all the aspects of the bear and it will do the aspect checks __it can__.